frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -157,11 +163,16 @@ const getCellHeader = (heading: string, index: number): ReactNode => {
   );
 };
 
-export function LineChartTable({ data }: Readonly<TableProps>) {
+export function LineChartTable({
+  data,
+  englandBenchmarkData,
+}: Readonly<TableProps>) {
   const tableData = mapToTableData(data);
-  const dataSortedByPeriodDesc = tableData.toSorted(
-    (a, b) => b.period - a.period
-  );
+  const englandData = englandBenchmarkData
+    ? mapToTableData(englandBenchmarkData)
+    : [];
+  const dataSortedByPeriodDesc = sortPeriod(tableData);
+  const englandRowDataDesc = sortPeriod(englandData);
```

Comment:
These variable names are confusing now:
```suggestion
  const dataSortedByPeriod = sortPeriod(tableData);
  const englandRowData = sortPeriod(englandData);
```